src/ansys/acp/core/_tree_objects/analysis_ply.py

```diff
@@ -112,6 +114,7 @@ def _create_stub(self) -> analysis_ply_pb2_grpc.ObjectServiceStub:
     status = grpc_data_property_read_only("properties.status", from_protobuf=status_type_from_pb)
     material = grpc_link_property_read_only("properties.material")
     angle: ReadOnlyProperty[float] = grpc_data_property_read_only("properties.angle")
+    thickness: ReadOnlyProperty[float] = grpc_data_property_read_only("properties.thickness")
```

Comment:
Add `supported_since="25.1"`. The `supported_since` keyword is added in the unit system PR https://github.com/ansys/pyacp/pull/567/files#diff-63421c0d3630c3f2a54ad926b024ab545ec5b423ef82c5f0d11999e30cda5a4fR300

Reply:
This is only a change in the frontend. The proto file and server did not change.